app/controllers/grades_controller.rb

```
@@ -273,10 +273,25 @@ def get_our_scores_data(team)
         reviews_of_our_work = get_heatgrid_data_for(reviews_of_our_work_maps)
         avg_score_of_our_work = team.aggregate_review_grade
 
+        # Embed all team metadata directly in this response so the frontend can populate
+        # team name, grade, submission links, and member list in a single API call.
+        # Previously the frontend made 3+ follow-up requests:
+        #   GET /participants/user/:id  →  GET /teams_participants/:id/list_participants
+        #   →  GET /users/:id  (once per team member)
+        # team.hyperlinks calls YAML.safe_load(submitted_hyperlinks) internally — no manual parsing.
+        # team.users is a single JOIN through teams_participants (no N+1 — one query for all members).
+        team_members = {
+            team_name:        team.name,
```

RuboCop (1.87.0)

[convention] Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

(Layout/FirstHashElementIndentation)